lib/api/core/storage/bootstrap/internalIndexBootstrap.js

```diff
@@ -190,6 +195,27 @@ class InternalIndexBootstrap extends SafeBootstrap {
 
     return seed;
   }
+
+  /**
+   * Loads authentication token from API key into Redis
+   *
+   * @returns {Promise}
+   */
+  async _loadApiKeys () {
+    // @todo test unit
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)